ui/raidboss/data/07-dt/ultimate/futures_rewritten.ts

```diff
@@ -12,11 +12,92 @@ const triggerSet: TriggerSet<Data> = {
   triggers: [],
   timelineReplace: [
     {
-      locale: 'en',
-      replaceText: {
+      'locale': 'en',
+      'replaceText': {
```

Comment:
Good catch, I totally forgot that we quote properties in the `timelineReplace` object when I wrote this.

Reply:
this is just pure luck that my script did this xD